File: lib/src/methods.dart
class Methods {
  /// Player related methods
  static const String createPlayer = 'createPlayer';
  static const String loadWithSourceConfig = 'loadWithSourceConfig';
  static const String loadWithSource = 'loadWithSource';
  static const String play = 'play';
  static const String pause = 'pause';
  static const String mute = 'mute';
  static const String unmute = 'unmute';
  static const String seek = 'seek';
  static const String currentTime = 'currentTime';
  static const String duration = 'duration';
  static const String destroy = 'destroy';
  static const String setTimeShift = 'setTimeShift';
  static const String getTimeShift = 'getTimeShift';
  static const String maxTimeShift = 'maxTimeShift';
  static const String isLive = 'isLive';
  static const String isPlaying = 'isPlaying';

  /// Player view related methods
  static const String destroyPlayerView = 'destroyPlayerView';
  static const String enterFullscreen = 'enterFullscreen';
  static const String exitFullscreen = 'exitFullscreen';

  /// Fairplay DRM related methods
  static const String fairplayPrepareMessage = 'fairplayPrepareMessage';
  static const String fairplayPrepareContentId = 'fairplayPrepareContentId';
  static const String fairplayPrepareCertificate = 'fairplayPrepareCertificate';
  static const String fairplayPrepareLicense = 'fairplayPrepareLicense';
  static const String fairplayPrepareLicenseServerUrl =
      'fairplayPrepareLicenseServerUrl';
  static const String fairplayPrepareSyncMessage = 'fairplayPrepareSyncMessage';

  /// Widevine DRM related methods
  static const String widevinePrepareMessage = 'widevinePrepareMessage';
  static const String widevinePrepareLicense = 'widevinePrepareLicense';
}

File: android/src/main/kotlin/com/bitmovin/player/flutter/Methods.kt
package com.bitmovin.player.flutter

class Methods {
    companion object {
        // Player related methods
        const val CREATE_PLAYER = "createPlayer"
        const val LOAD_WITH_SOURCE_CONFIG = "loadWithSourceConfig"
        const val LOAD_WITH_SOURCE = "loadWithSource"
        const val PLAY = "play"
        const val PAUSE = "pause"
        const val MUTE = "mute"
        const val UNMUTE = "unmute"
        const val SEEK = "seek"
        const val CURRENT_TIME = "currentTime"
        const val DURATION = "duration"
        const val DESTROY = "destroy"
        const val SET_TIME_SHIFT = "setTimeShift"
        const val GET_TIME_SHIFT = "getTimeShift"
        const val MAX_TIME_SHIFT = "maxTimeShift"
        const val IS_LIVE = "isLive"
        const val IS_PLAYING = "isPlaying"
        const val SEND_CUSTOM_DATA_EVENT = "sendCustomDataEvent"

        // Player view related methods
        const val DESTROY_PLAYER_VIEW = "destroyPlayerView"
        const val ENTER_FULLSCREEN = "enterFullscreen"
        const val EXIT_FULLSCREEN = "exitFullscreen"

        // Widevine DRM related methods
        const val WIDEVINE_PREPARE_MESSAGE = "widevinePrepareMessage"
        const val WIDEVINE_PREPARE_LICENSE = "widevinePrepareLicense"
    }
}
